.github/packaging/validate_build_output.sh

```diff
@@ -1,6 +1,20 @@
 package_type=${1}
-git clone -b v0.8.23 --depth=1  https://github.com/citusdata/tools.git tools
+
+# Since $HOME is set in GH_Actions as /github/home, pyenv fails to create virtualenvs.
+# For this script, we set $HOME to /root and then set it back to /github/home.
+GITHUB_HOME="${HOME}"
+export HOME="/root"
+
+eval "$(pyenv init -)"
+pyenv versions
+pyenv virtualenv ${PACKAGING_PYTHON_VERSION} packaging_env
+pyenv activate packaging_env
+
+git clone -b v0.8.24 --depth=1  https://github.com/citusdata/tools.git tools
 python3 -m pip install -r tools/packaging_automation/requirements.txt
 python3 -m tools.packaging_automation.validate_build_output --output_file output.log \
                                                             --ignore_file .github/packaging/packaging_ignore.yml \
                                                             --package_type ${package_type}
+pyenv deactivate
```

Comment:
no need to deactivate it here

Reply:
I added for the sake of completeness and understandability. Anyway I can remove as you said since it will automatically deactivated after step executio completed